README.md

```diff
@@ -1,160 +1,718 @@
+
+---
+
+#### PagingInfo
+
+Paging details, like "1 - 20 of 100 results".
+
+##### Example
+
+```jsx
+
+import { PagingInfo } from "@elastic/react-search-ui";
+
+...
+
+<PagingInfo />
+```
+
+##### Properties
+
+| Name | type      | Required? | Default                                                  | Options | Description                                                                                                                                         |
+| ---- | --------- | --------- | -------------------------------------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| view | Component | no        | [PagingInfo](packages/react-search-ui-views/src/PagingInfo.js) |         | Used to override the default view for this Component. See [Customization: Component views and HTML](#component-views-and-html) for more information. |
+
+---
+
+#### ErrorBoundary
+
+Handle unexpected errors.
+
+##### Example
+
+```jsx
+import { ErrorBoundary } from "@elastic/react-search-ui";
+
+...
+
+<ErrorBoundary>
+  <div>Some Content</div>
+</ErrorBoundary>
+```
+
+##### Properties
+
+| Name     | type       | Required? | Default                                                        | Options | Description                                                                                                                                         |
+| -------- | ---------- | --------- | -------------------------------------------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| children | React node | yes       |                                                                |         | Content to show if no error has occurred, will be replaced with error messaging if there was an error.                                              |
+| view     | Component  | no        | [ErrorBoundary](packages/react-search-ui-views/src/ErrorBoundary.js) |         | Used to override the default view for this Component. See [Customization: Component views and HTML](#component-views-and-html) for more information. |
+
+---
+
+### 4. Styles and Layout
+
+For basic styles, include:
+
+```jsx
+import "@elastic/react-search-ui-views/lib/styles/styles.css";
+```
+
+For a basic layout, which helps quickly get a UI bootstrapped, use the [Layout](packages/react-search-ui-views/src/layouts/Layout.js) Component.
+
+```jsx
+import { Layout } from "@elastic/react-search-ui-views";
+
+<Layout header={<SearchBox />} bodyContent={<Results />} />;
+```
+
+The provided styles and layout can be found in the [react-search-ui-views](packages/react-search-ui-views) package.
+
+For more information on customizing styles, see the [Customization: Custom Styles and Layout](#custom-styles-and-layout).
+
+## Configuration
```

Comment:
This comprehensive Configuration reference seems like something else that could be pulled out. You could probably simply expand the `SearchProvider` section to include a section or two about configuration and link out to a more comprehensive guide.